pkg/utils/util.go

```diff
@@ -106,7 +106,8 @@ const (
 	VolumeSnapshotContentInvalidLabel = "snapshot.storage.kubernetes.io/invalid-snapshot-content-resource"
 	// VolumeSnapshotInvalidLabel is applied to invalid snapshot as a label key. The value does not matter.
 	// See https://github.com/kubernetes/enhancements/blob/master/keps/sig-storage/177-volume-snapshot/tighten-validation-webhook-crd.md#automatic-labelling-of-invalid-objects
-	VolumeSnapshotInvalidLabel = "snapshot.storage.kubernetes.io/invalid-snapshot-resource"
+	VolumeSnapshotInvalidLabel          = "snapshot.storage.kubernetes.io/invalid-snapshot-resource"
+	VolumeSnapshotContentManagedByLabel = "snapshot.storage.kubernetes.io/managed-by"
```

Comment:
Can you add a comment to explain what this label is for?

Reply:
Please add something like this to clarify what is in this label: `It specifies the node name which handles the snapshot for the volume local to that node.`

Reply:
Done